src/version.h

```diff
@@ -35,4 +36,26 @@
 
 #define REDISGEARS_MODULE_NAME "rg"
 
-#endif /* SRC_VERSION_H_ */
+typedef struct RedisVersion{
```

Comment:
Why this was moved here?

Reply:
It's version-related, seems logical. Do you prefer it like it was?

Reply:
I see this file more as holding constants of the RedisGears version. This is used as Redis version and potentially more commonly used, this is why I believe it should be on common.

Reply:
no problem, will move it back.